source/core/Events.cpp

```diff
@@ -829,7 +829,7 @@ VIREO_FUNCTION_SIGNATURE2(UnregisterForEvents, RefNumVal, ErrorCluster)
 
 //------------------------------------------------------------
 struct WaitForEventArgs {
-    UInt32 *eventSpecIndex;
+    IntIndex *eventSpecIndex;
```

Comment:
Leave this as it was before (UInt32)

Reply:
See my next comment for why I would prefer to keep this as a signed int.